packages/compile-solidity/scripts/test.sh

```diff
@@ -3,8 +3,8 @@
 set -o errexit
 
 if [ "$CI" = true ]; then
-  mocha ./test/** ./test/**/* --timeout 10000 $@
+  mocha ./test/** ./test/**/* --timeout 50000 $@
```

Comment:
After making some of these changes, let's see if we can get by with removing this timeout increase.